package.json

```diff
@@ -19,6 +19,9 @@
     "presets": [
       "@babel/preset-env",
       "@babel/preset-react"
+    ],
+    "plugins": [
+      "@babel/plugin-proposal-class-properties"
```

Comment:
I could be wrong, but not sure this is needed tbh. (probs got that error for the missing pros. import in your component)